src/config/entities/configuration.ts

```diff
@@ -98,6 +98,11 @@ export default () => ({
           highRefreshRateTokensTtlSeconds: parseInt(
             process.env.HIGH_REFRESH_RATE_TOKENS_TTL_SECONDS ?? `${30}`,
           ),
+          // Public API has no explicit limit but can be rate limited/timeout
+          // Pro API has a limit of 100 addresses per request
```

Comment:
These values are not explicitly documented and were obtained by contacting support.